servers/src/main/java/tachyon/worker/block/evictor/EvictionPlan.java

```diff
@@ -44,6 +44,16 @@ public EvictionPlan(List<Pair<Long, BlockStoreLocation>> toTransfer, List<Long>
     return mToEvict;
   }
 
+  /**
+   * Whether the plan is empty, an empty plan means it is not null and both toMove and toEvict are
```

Comment:
Actually, I think it is helpful to also javadoc here that, an empty plan indicates no action (move or evict) needs to be taken to meet the requirement.